mlflow/server/js/src/components/App.css

```diff
@@ -175,6 +169,13 @@ table th {
   margin-left: 16px;
 }
 
+.truncate-text {
+  height: 1.3em;
```

Comment:
Hm, I don't think this height property makes sense here. Not all truncated text should have this height right?